pubsub/gossipsub/README.md

```diff
@@ -265,13 +268,16 @@ it publishes the message:
   source of the message.
 
 After processing the payload, it then processes the control messages in the envelope:
-- On `GRAFT(topic)` it adds the peer to `mesh[topic]` if it is
-  subscribed to the topic. If it is not subscribed, it responds with a `PRUNE(topic)`
-  control message.
-- On `PRUNE(topic)` it removes the peer from `mesh[topic]`.
-- On `IHAVE(ids)` it checks the `seen` set and requests unknown messages with an `IWANT`
+- On `GRAFT(topic, peer)` it adds a remote peer to `mesh[topic]` if it is
+  subscribed to the topic. If it is not subscribed, the remote peer responds with a
+  `PRUNE(topic)` control message. If the topic is not in the local peer's mesh then
+  it returns an error, the peer must first join the topic. (The remote peer must be in
+  the connected peers of the local peer, in order to check whether the remote is
+  subscribed to the topic.)
```

Comment:
there is no error returned here.
Also, I don't like this proliferation of parenteses.